src/main/java/seedu/expenditure/BorrowExpenditure.java

```diff
@@ -29,10 +30,15 @@ public void setBorrowerName(String borrowerName) {
         this.borrowerName = borrowerName;
     }
 
+    public String getFullDeadline() {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d MMM yyyy");
+        return getDeadline().format(formatter);
+    }
+
```

Comment:
This makes the date more intuitive and natural compared to our previous form. Good change!